README.md

```diff
@@ -16,29 +15,29 @@ To read more about code development and testing please read
 as well as the
 [FAQ on the project site](http://vmware.github.io/docker-volume-vsphere/documentation/faq.html).
 
-## Detailed documentation
+## Product documentation
+1. [VMware OSS Projects Page](http://vmware.github.io/docker-volume-vsphere/documentation/).
+2. [StorageHub](https://storagehub.vmware.com/#!/vsphere-storage/project-hatchway)
 
-Detailed documentation can be found on our [GitHub Documentation Page](http://vmware.github.io/docker-volume-vsphere/documentation/).
+## Downloads
 
-## Download
+**Releases are available on [Github](https://github.com/vmware/docker-volume-vsphere/releases), [Docker store](https://store.docker.com/plugins/vsphere-docker-volume-service) and [Bintray](https://bintray.com/vmware/product/vDVS/view)**
```

Comment:
I think we should just keep only Github releases link only as docker-store and bintray links are redundant.

Reply:
There is no place where we call out all the places customers need to go to in order to get vDVS. So I prefer to keep it.

Reply:
We have decided to stick with "**Releases are available on [Github](https://github.com/vmware/docker-volume-vsphere/releases)**" to have central place of the information for the releases. 

Moreover, we have badge icon too for all the releases; this is totally unnecessary and redundant to me. It is good to have less verbose. 

My vote is on not adding this extra verbose. 

/CC @pdhamdhere

Reply:
Yes. No point in duplicating what's already their on GitHub Releases page. I would simply redirect to "GitHub Releases" page.